clients/clearly-defined/src/main/kotlin/Enums.kt

```diff
@@ -31,28 +31,40 @@ import kotlinx.serialization.json.jsonPrimitive
 enum class ComponentType {
     @SerialName("npm")
     NPM,
+
```

Comment:
> all these empty lines are bad for overview.

Are they? Actually I believe they highlight nicely that an annotation belong to the line after, not top the line before.

> be consistent with data class constructors where it seems no rule complains about it

You managed to pick the OSV client as an example, which is practically the only place where we don't already use empty lines before annotations 😉 

> I preferred to write this in one line like below for overview

Obviously it boils down to a matter of taste; I feel this worsens overview as not all `val`s start in the same column.

Reply:
In my case I have a class with many properties and not all require the annotation, I found it a lot better to have one property per line and see the whole file at once, instead having empty lines between some properties that have the annotation, but not between that don't need the annotation, and ultimately have to scroll to see the whole file. Also, these serialization models are trivial code. Compare these two (23 vs. 46 lines):

```kotlin
@Serializable
internal data class PythonInspectorPackage(
    val type: String,
    val namespace: String?,
    val name: String,
    val version: String,
    val description: String,
    val parties: List<PythonInspectorParty>,
    @SerialName("homepage_url") val homepageUrl: String,
    @SerialName("download_url") val downloadUrl: String,
    val size: Long,
    val sha1: String?,
    val md5: String?,
    val sha256: String?,
    val sha512: String?,
    @SerialName("code_view_url") val codeViewUrl: String?,
    @SerialName("vcs_url") val vcsUrl: String?,
    val copyright: String?,
    @SerialName("license_expression") val licenseExpression: String?,
    @SerialName("declared_license") val declaredLicense: String,
    @SerialName("source_packages") val sourcePackages: List<String>,
    @SerialName("repository_homepage_url") val repositoryHomepageUrl: String?,
    @SerialName("repository_download_url") val repositoryDownloadUrl: String?,
    @SerialName("api_data_url") val apiDataUrl: String,
    val purl: String
)
```

```kotlin
@Serializable
internal data class PythonInspectorPackage(
    val type: String,
    val namespace: String?,
    val name: String,
    val version: String,
    val description: String,
    val parties: List<PythonInspectorParty>,
    
    @SerialName("homepage_url")
    val homepageUrl: String,
    
    @SerialName("download_url")
    val downloadUrl: String,
    
    val size: Long,
    val sha1: String?,
    val md5: String?,
    val sha256: String?,
    val sha512: String?,
    
    @SerialName("code_view_url")
    val codeViewUrl: String?,
    
    @SerialName("vcs_url")
    val vcsUrl: String?,
    
    val copyright: String?,
    
    @SerialName("license_expression")
    val licenseExpression: String?,
    
    @SerialName("declared_license")
    val declaredLicense: String,
    
    @SerialName("source_packages")
    val sourcePackages: List<String>,
    
    @SerialName("repository_homepage_url")
    val repositoryHomepageUrl: String?,
    
    @SerialName("repository_download_url")
    val repositoryDownloadUrl: String?,
    
    @SerialName("api_data_url")
    val apiDataUrl: String,
    
    val purl: String
)
```

Reply:
> Compare these two (23 vs. 46 lines):

I'm sorry to admit, but I still prefer the 46 lines... 😁

Reply:
So how can we move forward @mnonnenmacher? Should I just disable this inspection?

Reply:
I agree with @mnonnenmacher, but not because of the line count.
For me, having empty lines for the properties with an annotation implies a separate context for the ones which are not separated with an empty line.

However, I'm ok with both proposals.

Reply:
> However, I'm ok with both proposals.

As you still didn't approve I assume there are yet reservations, so I've dropped that commit. I'll disable that inspection before upgrading Detekt in another PR.